lib/cli/run-option-metadata.js

```diff
@@ -114,3 +105,23 @@ const ALL_MOCHA_FLAGS = Object.keys(TYPES).reduce((acc, key) => {
 exports.isMochaFlag = flag => {
   return ALL_MOCHA_FLAGS.has(flag.replace(/^--?/, ''));
 };
+
+/**
+ * Returns expected yarg option type for a given mocha flag.
+ * @param {string} flag - Flag to check (can be with out without leading "--"")
+ * @returns {string | undefined} - If flag is a valid mocha flag, the expected type of argument for this flag is returned, otherwise undefined is returned.
+ */
```

Comment:
This shouldn't be mentioned in the docs site, so:

```suggestion
 * @returns {string | undefined} - If flag is a valid mocha flag, the expected type of argument for this flag is returned, otherwise undefined is returned.
 * @private
 */
```

And 🤷 cool hey this fixes the docs generator for me locally! Must be that Mocha's docdash version is so old it can't really parse TS syntax normally. Yet another motivation for #5207.